src/NLog/Fluent/LogBuilder.cs

```diff
@@ -279,17 +279,25 @@ public LogBuilder StackTrace(StackTrace stackTrace, int userStackFrame)
             return this;
         }
 
-#if NET4_5
         /// <summary>
         /// Writes the log event to the underlying logger.
         /// </summary>
         /// <param name="callerMemberName">The method or property name of the caller to the method. This is set at by the compiler.</param>
         /// <param name="callerFilePath">The full path of the source file that contains the caller. This is set at by the compiler.</param>
         /// <param name="callerLineNumber">The line number in the source file at which the method is called. This is set at by the compiler.</param>
         public void Write(
-            [CallerMemberName]string callerMemberName = null,
-            [CallerFilePath]string callerFilePath = null,
-            [CallerLineNumber]int callerLineNumber = 0)
+#if NET4_5
+            [CallerMemberName]
+#endif
+            string callerMemberName = null,
+#if NET4_5
+            [CallerFilePath]
+#endif
+            string callerFilePath = null,
+#if NET4_5
```

Comment:
maybe it was to long ago, but why only .net 4.5?

Reply:
Not sure I understand. NET4_5 means all newer versions (including NetStandard etc.)

Reply:
OK, but it wasn't there before. So why the change?

Reply:
Fixing  #2059 by just moving `NET4_5` (Has been there all the time)